documentation/app/template-remote-cluster.md

```diff
@@ -0,0 +1,201 @@
+
+---
+
+## Prerequisites
+- Two OpenShift clusters:
+  - Host cluster: runs RHDH, Argo CD, Tekton
+  - Remote cluster: receives the application deployment
+- CLI access: `oc`/`kubectl` authenticated to both clusters, `ArgoCD` CLI
+- Versions: RHDH 1.6+ with OCP 4.17
+- If deploying RHOAI-based apps on the remote cluster, install:
+  - Node Feature Discovery (with a `NodeFeatureDiscovery` CR)
+  - NVIDIA GPU Operator (with a `ClusterPolicy` CR)
+  - OpenShift AI Operator (with a `DataScienceCluster` CR)
+
+---
+
+## Step 1: Expose the remote cluster to RHDH UI via dynamic plugins (Topology/Kubernetes)
+
+Before installing/configuring RHDH using the ai-rhdh-installer, add the remote cluster to the installer’s dynamic plugin configuration so the Topology and Kubernetes plugins can read it.
+
+On the remote cluster, create a service account and bind permissions:
+
+#### Create a service account token
+
+
+Service account token to view:
+
+- Kubernetes: Manually create a Secret for a Service Account — `https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-secret-for-a-service-account`
+- Backstage: Kubernetes configuration — `https://backstage.io/docs/next/features/kubernetes/configuration/`
+
+
+```bash
+oc create serviceaccount rhdh-sa -n default
+
+# Grant read-only access cluster-wide
+oc create clusterrolebinding rhdh-sa-view \
+  --clusterrole=view \
+  --serviceaccount=default:rhdh-sa
+
+# Create a Secret that yields a view token
+cat <<'EOF' | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rhdh-sa-token
+  namespace: default
+  annotations:
+    kubernetes.io/service-account.name: rhdh-sa
+type: kubernetes.io/service-account-token
+EOF
+
+# Retrieve and print the token
+oc get secret rhdh-sa-token -n default -o jsonpath='{.data.token}' | base64 --decode
+```
+
+Add a cluster entry to the installer’s plugin config. Example shapes for cluster locator configuration:
+
+Minimal entry (used by some dynamic plugins):
+
+```yaml
+- authProvider: serviceAccount
+  name: <remote-cluster-name>
+  serviceAccountToken: <paste-token-here>
+  skipTLSVerify: true
+  url: https://api.<your-remote-cluster>.
+```
+
+Full Backstage Kubernetes backend plugin config using `clusterLocatorMethods`:
+
+```yaml
+pluginConfig:
+  kubernetes:
+    serviceLocatorMethod:
+      type: multiTenant
+    clusterLocatorMethods:
+      - type: config
+        clusters:
+          - url: https://kubernetes.default.svc
+            name: <cluster-name>
+            authProvider: serviceAccount
+            serviceAccountToken: ${K8S_SA_TOKEN}
+            skipTLSVerify: true
+          # Your remote cluster info goes here
+    customResources:
+      - group: route.openshift.io
+        apiVersion: v1
+        plural: routes
+      - group: tekton.dev
+        apiVersion: v1
+        plural: pipelineruns
+      - group: tekton.dev
+        apiVersion: v1
+        plural: taskruns
+```
```

Comment:
I think I am wondering what the point of these instructions are in a techdoc 🤔

Reply:
To view this techdoc, I need to click here but RHDH is already installed:
<img width="404" height="435" alt="Screenshot 2025-10-02 at 8 05 08 PM" src="https://github.com/user-attachments/assets/3f6305d7-cfd2-40ae-80cf-b33498cb9c62" />

These instructions are for configuring RHDH but I am already viewing these when RHDH is installed and I may not have configured it for remote deployment:

<img width="1369" height="905" alt="Screenshot 2025-10-02 at 8 05 41 PM" src="https://github.com/user-attachments/assets/db5b8e09-6e28-48cb-b8ca-1b666fb092c8" />

Reply:
What do you think about techdocs should only guide how to deploy to remote i.e.;

1. provide remote cluster api
2. provide remote cluster deployment namespace 

nothing else.. we can discuss this with everyone else